### ISSUE TITLE
User Story: data analyst

### ISSUE THREAD

**Issue by drkane** [ThreeSixtyGiving/ID-reconcile#1]
*As a* data analyst using 360 giving data
*I want* duplicated organisations to have the same ID
*so that* my analysis doesn't double-count organisations

**Comment by morchickit** [comment]
Ok, when we say same, do we actually mean unique?

**Comment by drkane** [comment]
> Ok, when we say same, do we actually mean unique?

No, it's more that if we're referencing an organisation called "Brilliant Charity" and "Charity, Brilliant" that they both have the `GB-CHC-123456` identifier. Not sure how to describe it!
